safenode/src/bin/kadclient.rs

```diff
@@ -138,6 +145,42 @@ async fn wallet(opt: &Opt) -> Result<()> {
         }
     }
 
+    if let Some(hex) = &opt.send_to {
+        let address = parse_public_address(hex)?;
+
+        if let Some(amount) = opt
```

Comment:
Not sure if you're planning on erroring if the `send_amount` isn't given? I'm fine with this for now, but the user won't get to know if he specifies `send_to` but forgets the amount.